app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb

```diff
@@ -17,6 +17,8 @@ oses:
   - package_upgrade: boolean (default=true)
   - salt_master: string (default=undef)
   %>
+<%= snippet_if_exists(template_name + " custom early") %>
```

Comment:
we call this "custom pre" elsewhere, could we keep that for consistency?

Reply:
I chose "early" and "late" precisely to avoid "pre" and "post", because finish templates don't really have Pre and Post steps like kickstart templates do. But I'm not so strongly opinionated on this, so please let me know if this is a final decision and I will happily change the names.